### PR TITLE
Add Blog Post: Workflow Run Easy Filtering And Others

### PR DESCRIPTION
**Pull request by saraycp**

### CODE REVIEW

File: _posts/development/2024-07-04-workflow-run-easy-filtering-and-others.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Therefore, we let you decide what to do when you click on it.
+
+<figure>
+  <img src="/images/posts/sprint_168/filter_event_source.png" alt="Filter by event source when clicking on the value" />
+  <figcaption>Easy filtering by event source</figcaption>
+</figure>
+
+# Get Notified When A Token Is Shared With You
+
+Whoever sets up the SCM/CI Integration needs to create a workflow token in OBS. That person is also known as the *token executor* and they can share the token with other users or groups to let them see the workflow runs and manage the token on their behalf.
```
Comment (danidoni) [inline _posts/development/2024-07-04-workflow-run-easy-filtering-and-others.md:25]: This `token executor` thing is an internal naming convention, I think users won't care, so I would remove it.

```suggestion
Whoever sets up the SCM/CI Integration needs to create a workflow token in OBS. That person can share the token with other users or groups to let them see the workflow runs and manage the token on their behalf.
```

File: _posts/development/2024-07-04-workflow-run-easy-filtering-and-others.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+So far, when a workflow run failed, the executor received a notification. From now on, those users and groups they shared the token with will receive the notification as well.
```
Comment (danidoni) [inline _posts/development/2024-07-04-workflow-run-easy-filtering-and-others.md:27]: If accepting the suggestion above, then...

```suggestion
So far, when a workflow run failed, the workflow token creator received a notification. From now on, those users and groups they shared the token with will receive the notification as well.
```

File: _posts/development/2024-07-04-workflow-run-easy-filtering-and-others.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+# Documentation About GitHub Fine-grained Tokens
+
+GitHub currently supports two types of personal access tokens:
+
+- fine-grained personal access tokens
+- personal access tokens (classic)
+
+In OBS you can use any of them for the SCM/CI Integration. However, our documentation was not clear enough in this regard until now.
+Please read the new piece of documentation about [fine-grained personal access tokens](https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.scm_ci_workflow_integration#sec.obs.obs_scm_ci_workflow_integration.setup.token_authentication.how_to_authenticate_obs_with_scm),
```
Comment (danidoni) [inline _posts/development/2024-07-04-workflow-run-easy-filtering-and-others.md:37]: You know, ordering users around... :laughing: 

```suggestion
You can learn about that [here](https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.scm_ci_workflow_integration#sec.obs.obs_scm_ci_workflow_integration.setup.token_authentication.how_to_authenticate_obs_with_scm),
```

File: _posts/development/2024-07-04-workflow-run-easy-filtering-and-others.md (hunk 1 above)
Comment (saraycp) [reply _posts/development/2024-07-04-workflow-run-easy-filtering-and-others.md:25]: Good point!

File: _posts/development/2024-07-04-workflow-run-easy-filtering-and-others.md (hunk 2 above)
Comment (saraycp) [reply _posts/development/2024-07-04-workflow-run-easy-filtering-and-others.md:27]: Sure!

File: _posts/development/2024-07-04-workflow-run-easy-filtering-and-others.md (hunk 3 above)
Comment (saraycp) [reply _posts/development/2024-07-04-workflow-run-easy-filtering-and-others.md]: @danidoni  I rephrased this a bit, please have another look

**Review by danidoni — APPROVED** [review]

**Pull request merged**